cr.yaml

```diff
@@ -0,0 +1,4 @@
+# See https://github.com/helm/chart-releaser-action
```

Comment:
I think this file is unnecessary since these values are automatically detected by the action: https://github.com/helm/chart-releaser-action/blob/main/action.yml#L21-L22

And the `pages-branch` is `gh-pages` by default, which matches the github pages default branch which is used as the helm repository. Any reason why it is not `gh-pages` here?

Reply:
It's a personal preference to convey more semantic meaning rather than the general name of `gh-pages`. Happy to change since there is enough context given this repository is solely dedicated to Helm charts.